src/PrestaShopBundle/Form/Admin/Type/CustomMoneyType.php

```diff
@@ -27,11 +27,18 @@
 
 use Symfony\Component\Form\AbstractTypeExtension;
 use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Form\DataTransformerInterface;
 
-class CustomMoneyType extends AbstractTypeExtension
+class CustomMoneyType extends AbstractTypeExtension implements DataTransformerInterface
 {
     const PRESTASHOP_DECIMALS = 6;
 
+    public function buildForm(FormBuilderInterface $builder, array $options)
```

Comment:
Please add a PhpDoc header on your new methods.

It is important that each method describes:
- A short summary of what it does
- Type (and if possible, a description) of each parameter
- Type and description of what it returns
- Exceptions thrown and when

If your method is an extension of a parent class' method, you can use `@inheritdoc`.

Documenting methods is important because
- It enables autocompletion by IDEs
- It eases code understanding

Example:
```php
   /**
     * Executes the current command.
     *
     * This method is not abstract because you can use this class
     * as a concrete class. In this case, instead of defining the
     * execute() method, you set the code to execute by passing
     * a Closure to the setCode() method.
     *
     * @param InputInterface  $input  An InputInterface instance
     * @param OutputInterface $output An OutputInterface instance
     *
     * @return null|int null or 0 if everything went fine, or an error code
     *
     * @throws LogicException When this abstract method is not implemented
     *
     * @see setCode()
     */
    protected function execute(InputInterface $input, OutputInterface $output)
    {
        // ...
    }
```

You can find out more about PhpDoc by reading the [proposed PSR-5 standard](https://github.com/php-fig/fig-standards/blob/master/proposed/phpdoc.md).